src/internal.rs

```diff
@@ -435,7 +435,7 @@ pub struct Rav1dFrameContext_lf {
     pub last_sharpness: c_int,
     pub lvl: [[[[u8; 2]; 8]; 4]; 8], /* [8 seg_id][4 dir][8 ref][2 is_gmv] */
     pub tx_lpf_right_edge: Vec<u8>,  /* len = h*2 */
-    pub cdef_line_buf: *mut u8,
+    pub cdef_line_buf: AlignedVec32<u8>, /* AlignedVec32<DynPixel> */
     pub lr_line_buf: *mut u8,
```

Comment:
Can you initialize the field with `.write`?

Reply:
Done. I also added initialization logic for the other `Vec` fields, since it didn't look like we were doing that for them.